test/ttt_test.exs

```diff
@@ -18,20 +18,35 @@ defmodule TTTTest do
     input4 = ",,"
     input5 = "10"
     input6 = "0"
-    input7 = "5"
+    input7 = "9"
```

Comment:
Each of these inputs test different things. You should refactor each of these into a separate test. You might have 14 different tests since you are testing 14 different validation cases.